inc/dbmysql.class.php

```diff
@@ -1686,4 +1678,73 @@ public static function isNameQuoted($value): bool {
       $quote = static::getQuoteNameChar();
       return is_string($value) && trim($value, $quote) != $value;
    }
+
+   /**
+    * Remove SQL comments
+    * © 2011 PHPBB Group
+    *
+    * @param string $output SQL statements
+    *
+    * @return string
+    */
+   public function removeSqlComments($output) {
+      $lines = explode("\n", $output);
+      $output = "";
+
+      // try to keep mem. use down
+      $linecount = count($lines);
+
+      $in_comment = false;
+      for ($i = 0; $i < $linecount; $i++) {
+         if (preg_match("/^\/\*/", $lines[$i])) {
```

Comment:
```suggestion
         if (preg_match("/^\s*\/\*/", $lines[$i])) {
```

Reply:
The method has been copied verbatim from the original code from phpbb